Assets/Scripts/Object Controllers/CardHandler.cs

```diff
@@ -0,0 +1,60 @@
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class CardHandler : MonoBehaviour, IDragHandler, IBeginDragHandler, IEndDragHandler
+{
+    [SerializeField]
+    private string cardName = "null";
+    [SerializeField]
+    private float cooldown = 15;
+    [SerializeField]
+    private CardManager manager;
+    private Image cardImage;
+    private Button cardButton;
+    private Vector2 cardOriginPos;
+    private RectTransform rectTransform;
+
+    private void Awake()
+    {
+        cardImage = GetComponent<Image>();
+        cardButton = GetComponent<Button>();
+        rectTransform = GetComponent<RectTransform>();
+        CardIsPlayable();
+    }
+
+    private void CardIsPlayable()
```

Comment:
I think `MakeCardPlayable` and `StartCardCooldown` would be better because those names indicate operations more clearly than the current names.